lib/rollbar/item.rb

```diff
@@ -136,11 +136,18 @@ def handle_too_large_payload(stringified_payload, final_payload, attempts)
       uuid = stringified_payload['data']['uuid']
       host = stringified_payload['data'].fetch('server', {})['host']
 
+      original_error = {
+        :message => message,
+        :exception => exception,
+        :configuration => configuration,
+        :uuid => uuid,
+        :host => host
+      }
```

Comment:
Is there any possibility that including this extra data could cause the failsafe to fail? e.g. could there be an error in serialization, could the payload become too large...

(IIRC that's why this data was omitted from the failsafe before.)

Reply:
The intent is that everything that might fail is wrapped in a rescue, and if it fails we send the error message string instead. Payload size seems like an unlikely issue for the data that is sent, but I've added a commit that does truncate the unknown-length strings, and pre-serializes the options object to ensure only simple strings are sent for final serialization.